Main readme describes releases (#22)
kyma-project/bundles: README.md

## Usage

> The `bundles` folder contains sources of bundles and index files. These files are available in [releases](https://github.com/kyma-project/bundles/releases). To learn more about the release process, read [this](releasing.md) document.

Reviewer: Please fix the broken link

> The `bundles` folder contains sources of bundles and *index* files. A bundle repository files are available in [releases](https://github.com/kyma-project/bundles/releases). The [Releasing](releasing.md) document describes the releasing processes.

Reviewer: ```suggestion
The `bundles` folder contains sources of bundles and index files. These files are available in [releases](https://github.com/kyma-project/bundles/releases). To learn more about the release process, read [this](releasing.md) document.
```

If you want to configure the Helm Broker to use different set of bundles, read the [Helm Broker configuration](https://github.com/kyma-project/kyma/blob/master/docs/service-brokers/docs/011-configuration-helm-broker.md) and configure the proper URL.
 
## Development 
 

> Develop on your own remote repository forked from the original repository described in [getting started](docs/get-started.md) document and create your own bundles with a structure described in the [Helm Broker bundles](https://github.com/kyma-project/kyma/blob/master/docs/service-brokers/docs/012-configuration-helm-broker-bundles.md) document. Read also the [`CONTRIBUTING.md`](CONTRIBUTING.md) document that includes the contributing rules specific for this repository.

Reviewer: ```suggestion
Develop your own remote `bundles` repository forked from the original repository. Read [this](docs/getting-started.md) document to learn how. On your fork, you can create your own bundles with a structure described in the [Helm Broker bundles](https://github.com/kyma-project/kyma/blob/master/docs/service-brokers/docs/012-configuration-helm-broker-bundles.md) document. Read also the [`CONTRIBUTING.md`](CONTRIBUTING.md) document that includes the contributing rules specific for this repository.
```
